blockbuster/lane.go

```diff
@@ -66,7 +66,7 @@ type (
 
 		// ProcessLaneBasic validates that transactions belonging to this lane are not misplaced
 		// in the block proposal.
-		ProcessLaneBasic(txs []sdk.Tx) error
+		ProcessLaneBasic(ctx sdk.Context, txs []sdk.Tx) error
```

Comment:
I'd like to see the lane interface / all interfaces in BB / POB fixed at some point. It's fine now (cuz no one integrated) but in the future minor changes to interfaces are not backwards compatible